html.go

```diff
@@ -34,6 +37,7 @@ const (
 	HTML_NOREFERRER_LINKS                      // only link with rel="noreferrer"
 	HTML_HREF_TARGET_BLANK                     // add a blank target
 	HTML_TOC                                   // generate a table of contents
+	HTML_TOC_MD                                // if generating table of contents, output MarkDown
```

Comment:
This library spells Markdown as "Markdown", not "MarkDown". This should be kept consistent.

It's not very clear what this flag does. Output Markdown instead of what?